pkg/plugins/policies/core/matchers/validators/validator.go

```diff
@@ -2,11 +2,18 @@ package validators
 
 import (
 	"fmt"
+	"regexp"
 
 	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
 	"github.com/kumahq/kuma/pkg/core/validators"
 )
 
+var (
+	nameCharacterSet     = regexp.MustCompile(`^[0-9a-z-_]*$`)
+	tagNameCharacterSet  = regexp.MustCompile(`^[a-zA-Z0-9.\-_:/]*$`)
+	tagValueCharacterSet = regexp.MustCompile(`^([a-zA-Z0-9.\-_:/]*|\*)$`)
```

Comment:
Should we make and use the ones:

https://github.com/kumahq/kuma/blob/573a9a5f8b569a6d5606bdab384156c34420e022/pkg/core/resources/apis/mesh/validators.go#L39-L41

More generally should we move this whole file in:

https://github.com/kumahq/kuma/blob/master/pkg/core/resources/apis/mesh/validators.go

or at least in `/resources/apis/` @bartsmykla what do you think?

Reply:
I like the idea to move it to https://github.com/kumahq/kuma/blob/master/pkg/core/resources/apis/mesh/validators.go